cleanup: remove import alias for fence library along with other cleanup

humblec opened the pull request:
cleanup: remove import alias for fence library along with other cleanup
    
this commit remove unneeded import alias of fence library from the
network_fence test. also this commit correct some typos in the source
code comments.

   
 Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

humblec commented:
/retest all

nixpanic commented on e2e/cephfs.go:
```diff
@@ -296,7 +296,7 @@ var _ = Describe("cephfs", func() {
 			})
 
-			By("checking nodeplugin deamonset pods are running", func() {
+			By("checking nodeplugin daemonset pods are running", func() {
```
this has nothing to do with the PR, nor with the commit?

humblec replied:
@nixpanic mentioned or done in this commit as part of cleanup mentioned in the same ?

humblec replied:
if you are mentioning it becuase of "cephfs", sure, let me split these commits.

humblec replied:
regardless, rearranged the commits a bit.. ptal.. thanks @nixpanic

nixpanic commented on deploy/rbd/kubernetes/csi-provisioner-rbac.yaml:
```diff
@@ -86,9 +86,6 @@ metadata:
   namespace: default
   name: rbd-external-provisioner-cfg
 rules:
-  - apiGroups: [""]
```
rbd can use a configmap for KMS settings?

humblec replied:
hmmm.. let me revisit this part then :)

nixpanic commented on e2e/pvc.go:
```diff
@@ -339,7 +339,7 @@ func getMetricsForPVC(f *framework.Framework, pvc *v1.PersistentVolumeClaim, t i
 		}
 		if stdOut == "" {
-			e2elog.Logf("no metrics received from kublet on IP %s", kubelet)
+			e2elog.Logf("no metrics received from kubelet on IP %s", kubelet)
```
lots of typo fixes in the wrong commit :sob:

humblec replied:
yeah.. splitting these into different commits looked ugly :) , so consolidated in one commit :)

nixpanic commented on internal/rbd/rbd_util_test.go:
```diff
@@ -297,7 +297,7 @@ func TestStrategicActionOnLogFile(t *testing.T) {
 			switch tt.args.logStrategy {
 			case "compress":
-				newExt := strings.Replace(tt.args.logFile, ".log", ".gz", -1)
+				newExt := strings.ReplaceAll(tt.args.logFile, ".log", ".gz")
```
why?

humblec replied:
reverted..

Madhu-1 commented on deploy/cephfs/kubernetes/csi-provisioner-rbac.yaml:
```diff
@@ -77,10 +77,6 @@ metadata:
   name: cephfs-external-provisioner-cfg
 rules:
-  # remove this once we stop supporting v1.0.0
-  - apiGroups: [""]
-    resources: ["configmaps"]
-    verbs: ["get", "list", "create", "delete"]
```
same changes are required in helm charts and for the rbd.

humblec replied:
I am keeping this PR for just cleanup commit and will put RBAC one different.. @Madhu-1

Madhu-1 reviewed (changes requested):
Good to have separate PRs for typos and fixing the RBAC. helps in reviewing the code.

Madhu-1 commented:
@humblec as you are already working on RBAC cleanup can you please also take care of https://github.com/ceph/ceph-csi/issues/3018?

humblec commented:
> @humblec as you are already working on RBAC cleanup can you please also take care of #3018?

yeah. I am splitting this PR , so should be able to consolidate there in the new one.

Madhu-1 commented:
> > @humblec as you are already working on RBAC cleanup can you please also take care of #3018?
> 
> yeah. I am splitting this PR , so should be able to consolidate there in the new one.

Thanks @humblec assigned #3018 to you

humblec committed:
cleanup: remove import alias for fence library

this commit remove unneeded import alias of fence library
from the network_fence test.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

humblec committed:
cleanup: correct typos in test description and source code

this commit correct typos in various places.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

Rakshith-R commented:
@mergifyio rebase
